hill_climbing:
  run: true
  iterations: 1000
  goal: 80

simulated_annealing:
  run: false
  temperature: 1 # 1000
  cooling_rate: 0.999
  absolute_temperature: 0.1 #0.00001
